alpha-build-machinery/make/targets/golang/build.mk

```diff
@@ -0,0 +1,12 @@
+self_dir :=$(dir $(lastword $(MAKEFILE_LIST)))
+
+build:
```

Comment:
we should be able to define the dependencies for this target... e.g. `build: $(shell git ls-files)`

Reply:
I know what you mean but go already caches it's inputs and to specify them properly is not an easy task (tree of your dependencies is not restricted to your repo)